app_flutter/lib/build_dashboard_page.dart

```diff
@@ -130,6 +131,89 @@ class BuildDashboardPageState extends State<BuildDashboardPage> {
     );
   }
 
+  List<PopupMenuEntry<String>> _getTaskKey(bool isDark) {
+    const List<String> taskStatuses = <String>[
```

Comment:
We can pull from `TaskBox.statusColors.keys` so new colors get added automatically.